docs/2023/BeckN/Backn Smart Policy Infrastructure/updates/2023-07-07.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Week 1
+author: Mohammad Sarfraz Alam
+---
```

Comment:
As mentioned in my previous comment, the same goes for this part also.
Update your name here.
Also make sure you do the changes, in every week